kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/BatchRestApi.java

```diff
@@ -43,6 +49,16 @@ public Batch createBatch(BatchRequest request) {
     return this.getClient().post(API_BASE_PATH, requestBody, Batch.class, client.getAuthHeader());
   }
 
+  public Batch createBatch(BatchRequest request, File resourceFile) {
+    MultipartEntityBuilder entityBuilder =
+        MultipartEntityBuilder.create().setCharset(StandardCharsets.UTF_8);
+    entityBuilder.addPart(
+        "batchRequest", new StringBody(JsonUtils.toJson(request), ContentType.APPLICATION_JSON));
+    entityBuilder.addPart("resourceFile", new FileBody(resourceFile));
+    return this.getClient()
+        .post(API_BASE_PATH, entityBuilder.build(), Batch.class, client.getAuthHeader());
```

Comment:
the whole usage of entityBuilder is `resourceFile` right ? I mean there is no other code except name or file...